src/backend/commands/explain.c

```diff
@@ -537,7 +537,10 @@ ExplainOnePlan(PlannedStmt *plannedstmt, IntoClause *into, ExplainState *es,
 	 * AS, we'd better use the appropriate tuple receiver.
 	 */
 	if (into)
+	{
 		dest = CreateIntoRelDestReceiver(into);
+		plannedstmt->intoClause = into;
```

Comment:
In the case of a regular CTAS, we overwrite the `intoClause` value assigned in `ExplainOneQuery`. Moreover, a copy is created in `ExplainOneQuery` for into. Why don't we copy in this case? Would it be better to move the assignment to a new location instead of duplicating it?

Reply:
This is just a backport. Would you have preferred this as the [original solution](https://github.com/arenadata/gpdb/pull/1197/commits/c3f059f79bc609db4e17c8efb55e85c233b9e76d)?

Reply:
> This is just a backport

Okay It may be too late to rework something.

But I still wonder why we don't copy as is done in other places.

Reply:
Maybe it was necessary to make a copy...